src/Tools/CoreFx.Tools/GenerateBindingRedirect.cs

```diff
@@ -94,14 +94,16 @@ public override bool Execute()
                 {
                     XElement codeBase = new XElement(ns + "codeBase",
                                                 new XAttribute("version", result.Version),
-                                                new XAttribute("href", Path.GetFullPath(assembly.ItemSpec)));
+                                                new XAttribute("href", Path.GetFileName(assembly.ItemSpec)));
```

Comment:
I think it is worth to try as the codebase mainly to set the path

---
In reply to: [110052491](https://github.com/dotnet/corefx/pull/17975#discussion_r110052491) [](ancestors = 110052491)